packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -86,8 +97,86 @@ export type CallControlOptions = {
    * @defaultValue true
    */
   screenShareButton?: boolean | { disabled: boolean };
+  /* @conditional-compile-remove-from(stable): custom button injection */
+  /**
+   * Inject custom buttons in the call controls.
+   *
+   * @beta
+   */
+  onFetchCustomButtonProps?: CustomCallControlButtonCallback[];
 };
 
+/* @conditional-compile-remove-from(stable): custom button injection */
+/**
+ * Placement for a custom button injected in the {@link CallControls}.
+ *
+ * 'first': Place the button on the left end (right end in rtl mode).
+ * 'afterCameraButton': Place the button on the right (left in rtl mode) of the camera button.
+ * ... and so on.
+ *
+ * It is an error to place the button in reference to another button that has
+ * been hidden via a {@link CallControlOptions} field.
+ *
+ * Multiple buttons placed in the same position are appended in order.
+ * E.g., if two buttons are placed 'first', they'll both appear on the left end (right end in rtl mode)
+ * in the order provided.
+ *
```

Comment:
should we `@remarks` what happens when when a button is disabled? e.g. if I remove participants button I assume it will be placed where the participants button _was_

Reply:
By removed, I guess you mean hidden via `CallControlOptions`? The remark above says that you cannot place it relative to a button that was hidden. So placement is always in relation to another button that is visible.

I think talking about "placed instead of hidden button" will add confusion because it relates the custom button placement to a button that was hidden.

Reply:
Ah yep that comment resolves this, missed it thanks!